platform/osx/os_osx.h

```diff
@@ -127,6 +127,7 @@ class OS_OSX : public OS_Unix {
 	bool zoomed;
 	bool resizable;
 	bool window_focused;
+	bool on_top;
```

Comment:
Maybe it should be initialized to `false`?

Reply:
It's initialized in `OS_OSX::initialize`.

Reply:
Ah indeed, I was fooled by GitHub which wrongly assigns it as part of a static method:
![Screenshot_20200815_170605](https://user-images.githubusercontent.com/4701338/90315183-a6923e80-df19-11ea-9b85-28d410d5225b.png)

Reply:
Added init to false to the constructor as well.